src/PointwiseFunctions/GeneralRelativity/Christoffel.cpp

```diff
@@ -7,13 +7,14 @@
 #include "DataStructures/Tensor/Tensor.hpp"
 #include "Utilities/GenerateInstantiations.hpp"
 
+namespace gr {
 template <size_t SpatialDim, typename Frame, IndexType Index, typename DataType>
-tnsr::abb<DataType, SpatialDim, Frame, Index> compute_christoffel_first_kind(
+tnsr::abb<DataType, SpatialDim, Frame, Index> christoffel_first_kind(
     const tnsr::abb<DataType, SpatialDim, Frame, Index>& d_metric) {
   auto christoffel =
       make_with_value<tnsr::abb<DataType, SpatialDim, Frame, Index>>(d_metric,
                                                                      0.);
-  constexpr auto dimensionality = index_dim<0>(christoffel);
+  const constexpr auto dimensionality = index_dim<0>(christoffel);
```

Comment:
turns out const is implied by constexpr for variables, so don't add it

Reply:
const constexper auto -> constexpr auto, completing the code -> review -> fix -> review -> revert cycle :)